contracts/contracts/strategies/NativeStaking/ValidatorAccountant.sol

```diff
@@ -0,0 +1,269 @@
+import { IWETH9 } from "../../interfaces/IWETH9.sol";
+
+/// @title Validator Accountant
+/// @notice Attributes the ETH swept from beacon chain validators to this strategy contract
+/// as either full or partial withdrawals. Partial withdrawals being consensus rewards.
+/// Full withdrawals are from exited validators.
+/// @author Origin Protocol Inc
+abstract contract ValidatorAccountant is ValidatorRegistrator {
+    /// @notice The minimum amount of blocks that need to pass between two calls to manuallyFixAccounting
+    uint256 public constant MIN_FIX_ACCOUNTING_CADENCE = 7200; // 1 day
+    /// @notice The maximum amount of ETH that can be staked by a validator
+    /// @dev this can change in the future with EIP-7251, Increase the MAX_EFFECTIVE_BALANCE
+    uint256 public constant MAX_STAKE = 32 ether;
+
+    /// @notice Keeps track of the total consensus rewards swept from the beacon chain
+    uint256 public consensusRewards;
+
+    /// @notice start of fuse interval
+    uint256 public fuseIntervalStart;
+    /// @notice end of fuse interval
+    uint256 public fuseIntervalEnd;
+    /// @notice last block number manuallyFixAccounting has been called
+    uint256 public lastFixAccountingBlockNumber;
+
+    uint256[49] private __gap;
+
+    event FuseIntervalUpdated(uint256 start, uint256 end);
+    event AccountingFullyWithdrawnValidator(
+        uint256 noOfValidators,
+        uint256 remainingValidators,
+        uint256 wethSentToVault
+    );
+    event AccountingValidatorSlashed(
+        uint256 remainingValidators,
+        uint256 wethSentToVault
+    );
+    event AccountingConsensusRewards(uint256 amount);
+
+    event AccountingManuallyFixed(
+        int256 validatorsDelta,
+        int256 consensusRewardsDelta,
+        uint256 wethToVault
+    );
+
+    /// @param _wethAddress Address of the Erc20 WETH Token contract
+    /// @param _vaultAddress Address of the Vault
+    /// @param _beaconChainDepositContract Address of the beacon chain deposit contract
+    /// @param _ssvNetwork Address of the SSV Network contract
+    /// @param _maxValidators Maximum number of validators that can be registered in the strategy
+    constructor(
+        address _wethAddress,
+        address _vaultAddress,
+        address _beaconChainDepositContract,
+        address _ssvNetwork,
+        uint256 _maxValidators
+    )
+        ValidatorRegistrator(
+            _wethAddress,
+            _vaultAddress,
+            _beaconChainDepositContract,
+            _ssvNetwork,
+            _maxValidators
+        )
+    {}
+
+    /// @notice set fuse interval values
+    function setFuseInterval(
+        uint256 _fuseIntervalStart,
+        uint256 _fuseIntervalEnd
+    ) external onlyGovernor {
+        require(
+            _fuseIntervalStart < _fuseIntervalEnd &&
+                _fuseIntervalEnd < 32 ether &&
+                _fuseIntervalEnd - _fuseIntervalStart >= 4 ether,
+            "incorrect fuse interval"
```

Comment:
In our existing code base, we use an initial uppercase letter in errors (unless talking about a function name).

For example:


<img width="422" alt="image" src="https://github.com/OriginProtocol/origin-dollar/assets/837/aaab0713-001a-4b65-9a4c-dc9abc99f178">